modules/pel/peltool/peltool.py

```diff
@@ -427,9 +459,19 @@ def main():
         else:
             if not os.path.isdir(args.list):
                 sys.exit(f"{args.list} is not a valid directory")
+            if args.pelID:
+                parsePelFromID(args.list, args.pelID, config)
+                sys.exit(0)
             listOption(args.list, config, args.extension)
```

Comment:
Would we generate a summary of PELs even if we utilized the --list and --id options outside of the BMC environment?

However, I believe we should only display information pertaining to the specified PEL ID.

Reply:
Pls ignore this comment, Its my bad I did not notice LN464